vignettes/sir_models.Rmd

```diff
@@ -0,0 +1,680 @@
+---
+title: "SIR models with odin, dust and mcstate"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{sir_models}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(odin.dust)
+library(mcstate)
+library(rmarkdown)
+```
+
```

Comment:
Probably we need some basic introductory stuff here about what the problem is we're trying to solve